config/i18n/locales/indonesian/serve.json

```
@@ -0,0 +1,3 @@
+{
+    "redirects": []
+}
```

```suggestion
}

```
EOF NL should fix linter

---

Awesome, thanks for catching that @naomi-lgbt.